src/dotnet-interactive/CommandLine/readme.md

```diff
@@ -9,21 +9,21 @@ A number of APIs are available over HTTP in `dotnet-interactive`. You can enable
 dotnet interactive http                                        | Enabled on auto-selected port
 dotnet interactive http --http-port *                          | Enabled on auto-selected port
 dotnet interactive http --http-port 1234                       | Enabled on port 1234
-dotnet interactive http --http-port-range 8001-9000            | Enabled on a port between 8001 and 9000
+dotnet interactive http --http-port-range 8001-9000            | UNSUPPORTED
```

Comment:
Not sure if you plan was to delete the ones that are unsupported, but I don't think documenting them is necessary.

Reply:
removing them